dozer/cogs/levels.py

```diff
@@ -427,14 +443,14 @@ async def _cfg_guild_setting(self, ctx, xp_min=None, xp_max=None, xp_cooldown=No
                     xp_cooldown=15,
                     entropy_value=0,  # Is in table but is not used yet
                     lvl_up_msgs=-1,
-                    enabled=True
+                    enabled=False
                 )
 
             ent = GuildXPSettings(
                 guild_id=int(ctx.guild.id),
                 xp_min=int(xp_min) if xp_min else old_ent.xp_min,
                 xp_max=int(xp_max) if xp_max else old_ent.xp_max,
-                xp_cooldown=int(xp_cooldown) if xp_cooldown else old_ent.xp_cooldown,
+                xp_cooldown=int(xp_cooldown) if xp_cooldown or xp_cooldown == 0 else old_ent.xp_cooldown,
```

Comment:
`... if xp_cooldown is not None else ...` more clearly communicates what this line does.